src/lto_cli/commands/transfer.py

```diff
@@ -4,7 +4,7 @@
 def func(name_space, parser):
 
     recipient = name_space.recipient[0]
-    amount = name_space.amount[0]
+    amount = name_space.amount[0] * 100000000
```

Comment:
Amount isn't rounded / cast to an int.

Reply:
i think i might have got it wrong, 
if i cast the amount to int ```int(name_space.amonut[0])``` we're going to loose the decimal points so i think you meant something else

Reply:
`amount = int(name_space.amount[0] * 100000000)` should be ok